js/views/messagesitem.js

```diff
@@ -98,6 +98,8 @@ define(function(require) {
 		openMessage: function(event) {
 			event.stopPropagation();
 			$('#mail-message').removeClass('hidden-mobile');
+			// make sure message is marked as read when clicked on it
+			Radio.ui.trigger('messagesview:messageflag:set', this.model.id, 'unseen', false);
```

Comment:
⚠️ This will conflict with the changes in https://github.com/nextcloud/mail/pull/326/commits/96bdcd77125b05c7baf5ff1656aeb30f5133aef4#diff-044c7b2c263cd5812964387e6c41c8fcR321. Git won't detect this, so we have to be careful with merging whichever PR is integrated last and adjust the code accordingly.